### PR TITLE
Add meili search

### PR DESCRIPTION
**Pull request by chripoh**

### CODE REVIEW

File: rotaract-club-finder.php [hunk 1]
```diff
@@ -35,14 +35,19 @@
...
  * The core plugin class that is used to define internationalization,
  * admin-specific hooks, and public-facing site hooks.
  */
 require plugin_dir_path( __FILE__ ) . 'includes/class-rotaract-club-finder.php';
 
+/**
+ * TEST
+ */
+//require plugin_dir_path( __FILE__ ) . 'includes/callers/meili.php';
+
```
Comment (bbenno) [inline rotaract-club-finder.php:50]: forgot to remove?

**Review by bbenno — CHANGES_REQUESTED** [review]
Following [semantic versioning](https://semver.org/), the next version will be 3.0.0 as we have some breaking changes. Could you update the version strings accordingly?

File: README.adoc
Comment (bbenno) [inline README.adoc:1]: Could you please add some explanation on how to configure, setup and use the new Meilisearch integration.

File: .gitignore
Comment (bbenno) [inline .gitignore:1]: strong recommendation: use an IDE or editor supporting [editorconfig](https://editorconfig.org/) and add a tailing newline to this file :smile:

File: includes/callers/class-rotaract-club-finder-meilisearch-caller.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+   * Set the Meilisearch host URL if defined.
+   *
+   * @since    2.2.6
+   */
+  public function __construct()
+  {
+    if (
+      defined('ROTARACT_MEILISEARCH_API_KEY')
+    ) {
+      $this->client =  new Client('https://search.rotaract.de', ROTARACT_MEILISEARCH_API_KEY);
```
[the remark this answers is not in the document]
Comment (chripoh) [reply includes/callers/class-rotaract-club-finder-meilisearch-caller.php:47]: We could define a constant (e.g. ROTARACT_MEILISEARCH_URL) in the wp-config, like the api key?

File: public/js/public.js [hunk 3]
```diff
@@ -98,14 +101,16 @@ function handleResults( data ) {
...
+	initMap( searchLocation, meili );
 }
 
 function searchClubs( event ) {
 	event.preventDefault();
 	const searchLocation = document.getElementById( 'rotaract-search' ).value;
 	const range          = document.getElementById( 'club-finder-range' ).value;
 
+	console.log( 'Searching for clubs in ' + range + 'km around ' + searchLocation + '.' );
+
```
Comment (bbenno) [inline public/js/public.js:113]: should this be included in production code?

File: README.adoc
Comment (bbenno) [inline README.adoc:1]: Could you remove all doc related to ElasticSearch (e.g. setting constants in wp-config.php)

**Review by bbenno — CHANGES_REQUESTED** [review]

**Review by bbenno — APPROVED** [review]

**Pull request merged**